runtime/src/bank.rs

```diff
@@ -1708,7 +1697,7 @@ impl Bank {
             fee_structure: FeeStructure::default(),
             #[cfg(feature = "dev-context-only-utils")]
             hash_overrides: Arc::new(Mutex::new(HashOverrides::default())),
-            accounts_lt_hash: Mutex::new(AccountsLtHash(LtHash([0xBAD2; LtHash::NUM_ELEMENTS]))),
+            accounts_lt_hash: Mutex::new(AccountsLtHash(LtHash([0xBAD1; LtHash::NUM_ELEMENTS]))),
```

Comment:
Is `0XBAD?` some sort of invalid marker for "BAD" LT hash? If so, can we make it a constant function?

Reply:
And what's the difference between "BAD1" and "BAD2"?